mne/evoked.py

```diff
@@ -1436,6 +1436,77 @@ def _write_evokeds(fname, evoked, check=True):
         end_file(fid)
 
 
+@fill_doc
+def export_evokeds(fname, evoked, fmt='auto', device=None, history=None):
+    """Export evoked dataset to external formats.
+
+    Supported formats: MFF (mff, uses module mffpy)
+
+    .. warning::
+        Since we are exporting to external formats, there's no guarantee that
+        all the info will be preserved in the external format. To save in
+        native MNE format (``.fif``) without information loss, use
+        :func:`mne.write_evokeds` instead.
+
+    Parameters
+    ----------
+    %(export_params_fname)s
+    evoked : Evoked instance, or list of Evoked instances
+        The evoked dataset, or list of evoked datasets, to export to one file.
+        Note that the measurement info from the first evoked instance is used,
+        so be sure that information matches.
+    fmt : 'auto' | 'mff'
+        Format of the export. Defaults to ``'auto'``, which will infer the
+        format from the filename extension. See supported formats above for
+        more information.
+    device : None (default) | str
```

Comment:
We have to be careful about future compat here. Each export could have zero or more such extra arguments, and I foresee a nasty parameter list if, say, one of these took 10 such arguments.

I don't see a great way around it (`**kwargs` seems worse for readability, and `mff_kwargs=None, eeglab_kwargs=None, ...` just punts on the issue. So how about we prefix the relevant exporter name:
```suggestion
    mff_device : None (default) | str
```
That way it will clearly group visually in the parameter list:
```
mff_device : ...
    ...
mff_history : ...
    ...
eeglab_foo : ...
    ...
brainvision_bar : ...
    ...
```

Reply:
Good call.